pom.xml

```diff
@@ -774,10 +774,6 @@
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <version>${jacoco.version}</version>
-                <configuration>
-                    <destfile>${basedir}/target/coverage-reports/jacoco-unit.exec</destfile>
-                    <datafile>${basedir}/target/coverage-reports/jacoco-unit.exec</datafile>
-                </configuration>
```

Comment:
Why we commented this?

Reply:
Intellij was highlighting it in red that "Element destfile/datafile is not allowed here" and coverage-reports doesn't exists in target folder, so those lines are not doing anything.

Reply:
Ok. It seems that these properties are `destFile` and `dataFile` but if they worked before with default values then good that we get rid of it